atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -196,11 +198,32 @@ def _should_send(self, rebuild, success, auto_canceled, manual_canceled):
             should_send |= should_send_mapping[state]
         return should_send
 
+    def _get_image_name_and_repos(self):
+
+        repos = []
+        stored_data = self.workflow.exit_results.get(StoreMetadataInOSv3Plugin.key)
+        if not stored_data or 'annotations' not in stored_data:
+            raise ValueError('Stored Metadata not found')
+
+        repo_data = json.loads(stored_data['annotations']['repositories'])
+
+        repos.extend(repo_data['unique'])
+        repos.extend(repo_data['primary'])
```

Comment:
why are we adding primary, if we not ever use it?

Reply:
what makes you say we don't use it?

Reply:
nvm I see it is used outside of method

Reply:
ok np